apps/mobile/src/features/Signers/components/SignersList/SignersList.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useMemo } from 'react'
+
+import { SafeListItem } from '@/src/components/SafeListItem'
+import { getTokenValue, Spinner } from 'tamagui'
+
+import { SectionList } from 'react-native'
+import { useCallback } from 'react'
+import { useScrollableHeader } from '@/src/navigation/useScrollableHeader'
+import { NavBarTitle } from '@/src/components/Title'
+import { SignersListHeader } from './SignersListHeader'
+import { SafeState } from '@safe-global/store/gateway/AUTO_GENERATED/safes'
+import { AddressInfo } from '@safe-global/store/gateway/AUTO_GENERATED/transactions'
+import SignersListItem from './SignersListItem'
+
+export type SignerSection = {
+  title: string
+  data: SafeState['owners']
+}
+
+const keyExtractor = (item: AddressInfo, index: number) => item.value + index
+
+interface SignersListProps {
+  signersGroup: SignerSection[]
+  isFetching: boolean
+  hasLocalSingers: boolean
+}
+
+export function SignersList({ signersGroup, isFetching, hasLocalSingers }: SignersListProps) {
+  const { handleScroll } = useScrollableHeader({
+    children: <NavBarTitle paddingRight={5}>Signers</NavBarTitle>,
```

Comment:
I don't think we need the paddingRight. It was there on the pending Tx screen because there we were rendering a the badge with the tx count.